rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -158,6 +161,19 @@ def test_curve_deposit(database, ethereum_transaction_decoder):
             sequence_index=371,
             timestamp=1650276061000,
             location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
```

Comment:
Question: In this test only the order of detected events changed, due to what you did, right? No other details changed, apart from the "extra_data" right?

Reply:
I've checked the tests and seems that the change is re order + adding the info in the extra_data field